src/modules/userMethods.ts

```diff
@@ -20,8 +21,13 @@ const create = async (data: CreateUser) => {
     return prisma.user.create({ data: { ...data } });
 };
 
-const find = async (id: number): Promise<User> => {
-    const user = await prisma.user.findFirst({ where: { id } });
+const find = async (id: number): Promise<User & WithSettings> => {
+    const user = await prisma.user.findFirst({
```

Comment:
Probably unnecesary. As far as I can tell you need settings only to get settingsId